### PR TITLE
clean-up(trusted,getter calls): remove raw payload signature support

### PR DESCRIPTION
**Pull request by jonalvarezz**
### Context

Raw signatures isn't really used and the shorter hashed version should be preffered instead.

This pull request remove the raw payload checks support and rework ts-tests to hash the payload or the prefix + hashed payload combo.

## TODO
* [x] drop raw payload verification support
* [x] update ts-tests
* [x] ts-tests: add prefix + hashed payload tests
* [x] test on IDH.

### CODE REVIEW

File: tee-worker/ts-tests/integration-tests/common/di-utils.ts [hunk 1]
```diff
@@ -45,6 +52,9 @@ async function sendRequest(
 
                 if (res.status.isTrustedOperationStatus && res.status.asTrustedOperationStatus[0].isInvalid) {
                     console.log('Rpc trusted operation execution failed, hash: ', res.value.toHex());
+                    const stfError = api.createType('StfError', res.value);
+                    const msg = stfErrorToString(stfError);
+                    console.log('TrustedOperationStatus error: ', msg);
```
Comment (0xverin) [inline tee-worker/ts-tests/integration-tests/common/di-utils.ts:57]: 3

File: tee-worker/ts-tests/integration-tests/di_substrate_identity.test.ts [hunk 2]
```diff
@@ -560,7 +567,13 @@ describe('Test Identity (direct invocation)', function () {
...
 
         // the third (last) identity in the IDGraph is eveSubstrateIdentity
-        assert.equal(idGraph[3][1].web3networks.toHuman()?.toString(), expectedWeb3Networks.toString());
+        const eveSubstrateIdentity = idGraph[3];
+        const [, { web3networks }] = eveSubstrateIdentity;
+        const expectedWeb3Networks = ['Polkadot', 'Litentry'];
+
+        assert.equal(web3networks.length, expectedWeb3Networks.length);
+        assert.equal(web3networks.indexOf('Polkadot') !== -1, true);
+        assert.equal(web3networks.indexOf('Litentry') !== -1, true);
```
Comment (0xverin) [inline tee-worker/ts-tests/integration-tests/di_substrate_identity.test.ts:574]: Forgot to uncomment?

**Review by 0xverin — COMMENTED** [review]
I only checked the ts-tests part, overall it looks good. Thank you for adding detailed logs of `stferror`. In fact, I was planning to do this because we had a related discussion before on the channel. :p

**Review by Kailai-Wang — COMMENTED** [review]
Thank you - I'm good to merge it once CI passes : )

File: tee-worker/ts-tests/integration-tests/di_substrate_identity.test.ts (hunk 2 above)
Comment (jonalvarezz) [reply tee-worker/ts-tests/integration-tests/di_substrate_identity.test.ts:574]: yes! 😂

Thanks, @0xverin

**Review by Kailai-Wang — APPROVED** [review]

**Pull request merged**